form/src/main/java/org/dhis2/form/data/FormRepositoryImpl.kt

```diff
@@ -463,7 +467,7 @@ class FormRepositoryImpl(
 
     override fun updateErrorList(action: RowAction) {
         if (action.error != null) {
-            if (itemsWithError.find { it.id == action.id } == null) {
+            if (itemsWithError.none { it.id == action.id }) {
```

Comment:
Have you verified this works as expected? I tried this in the enrollment repository and it caused a test to fail

Reply:
It didn't failed to me, 
That is the none function implementation:
```
/**
 * Returns `true` if no elements match the given [predicate].
 * 
 * @sample samples.collections.Collections.Aggregates.noneWithPredicate
 */
public inline fun <T> Iterable<T>.none(predicate: (T) -> Boolean): Boolean {
    if (this is Collection && isEmpty()) return true
    for (element in this) if (predicate(element)) return false
    return true
}
```